ospfd/ospf_ri.c

```diff
@@ -1511,22 +1615,26 @@ static uint16_t show_vty_sr_range(struct vty *vty, struct tlv_header *tlvh)
 }
 
 /* Display Segment Routing Maximum Stack Depth TLV information */
-static uint16_t show_vty_sr_msd(struct vty *vty, struct tlv_header *tlvh)
+static uint16_t show_vty_sr_msd(struct vty *vty, struct tlv_header *tlvh,
+				json_object *json)
 {
 	struct ri_sr_tlv_node_msd *msd = (struct ri_sr_tlv_node_msd *)tlvh;
 
 	check_tlv_size(RI_SR_TLV_NODE_MSD_SIZE, "Node Maximum Stack Depth");
 
-	if (vty != NULL) {
-		vty_out(vty,
-			"  Segment Routing MSD TLV:\n"
-			"    Node Maximum Stack Depth = %d\n",
-			msd->value);
-	} else {
+	if (vty != NULL)
+		if (!json)
+			vty_out(vty,
+				"  Segment Routing MSD TLV:\n"
+				"    Node Maximum Stack Depth = %d\n",
+				msd->value);
+		else
+			json_object_int_add(json, "nodeMaximumSIDdepth",
```

Comment:
According to our JSON convention it should be more like `nodeMaximumSidDepth`.

Reply:
Done